firmware/mods/ai_stackchan_api/README_ja.md

```
@@ -52,12 +52,12 @@
 ※日本語対応にする場合は"model"を"eleven_multilingual_v2"にしてください。
 
 ### 2. ビルド、書き込み
-次の手順でModを書き込みます。--targetは使用するデバイスを指定してください。
+次の手順でホストとModを書き込みます。
+M5Stack Core2 を使う場合は `:core2` と `core2` を指定します。
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)